view/template_form.xml

```diff
@@ -9,7 +9,12 @@ this repository contains the full copyright notices and license terms. -->
             <field name="products"/>
         </page>
         <page string="Description" id="template-description">
-            <field name="description"/>
+            <group col="2" colspan="6" yexpand="1" yfill="1" id="product_description">
+                <separator string="Long Description"/>
```

Comment:
build is failing, you can use `<separator name="long_description"/>` and string will come from field